client/landing/stepper/declarative-flow/internals/steps-repository/intent-step/index.tsx

```diff
@@ -53,7 +52,12 @@ const IntentStep: Step = function IntentStep( { navigation } ) {
 			goNext={ goNext }
 			skipLabelText={ translate( 'Skip to dashboard' ) }
 			skipButtonAlign={ 'top' }
-			hideBack={ ! ( isEnabled( 'signup/site-vertical-step' ) && isEnglishLocale ) }
+			hideBack={
+				! (
+					isEnabled( 'signup/site-vertical-step' ) &&
+					englishLocales.includes( translate.localeSlug || i18nDefaultLocaleSlug )
```

Comment:
Do you encounter any issues when using `useIsEnglishLocale`?

Reply:
Ops, I am sorry I am not sure how this change got in. Let me clear this up. 🙈